pptx/text/text.py

```diff
@@ -439,6 +439,47 @@ def underline(self, value):
         self._element.u = value
 
 
+    @property
+    def strikethrough(self):
+        """
+        Read/write.  |True|, |False|, |None|, or a member of ST_TextStripeType
+        which incluides (noStrike, sngStrike, dblStrike).  |None| is the default
+        and indicates the strikethrough setting should be inherited from the style
+        hierarchy, such as a placeholder.  |True| indicates a single strikethrough.
+        |False| indicates no strikethrough.  Other settings indicate a double or
+        single strikethrough.  
+        """
+        strike = self._rPr.strike
+        if strike == 'sngStrike':
+            return True
+        if strike == 'noStrike':
+            return False
+        return strike
+    
+    @strikethrough.setter
+    def strikethrough(self, value):
+        if value is True:
+            value = 'sngStrike'
+        elif value is False:
+            value = 'noStrike'
+        self._rPr.strike = value
```

Comment:
How come you didn't implement double strike through? What I'm understanding is that only `True` correlates to single strike through  and to set double strike through we need to set it to some other value that is not `True` or `False`. Can you expand what these other settings are?

Reply:
This lets us do it all.  You can simply pass True or False to turn it on/off.  You can pass None to let it inherit from the parent.  Additionally you can pass a specific value `noStrike`, `sngStrike`, or `dblStrike`.  This is essentially the exact same way that Underlines are handled.